### PR TITLE
Add LaTeX axes examples

### PR DESCRIPTION
**Pull request by ianthomas23**
This PR adds 2 new examples to show off the new LaTeX axis and tick labels. Output PNGs are
blackbody
normal
The first example is the "flashy, dense with color" one with LaTeX on both axis labels. The second one is the less flashy but complete one that has LaTeX on both axis labels and both sets of tick labels.

I'm submitting them as example scripts in case we want the python scripts to hang around, otherwise just the output PNGs may be sufficient. They can be improved in time when we extend the LaTeX functionality further, e.g. using the theme colors for LaTeX labels.

Thanks to both @tcmetzger and @IuryPiva for help with these.

Given that time is of the essence with regard to the 2.4 release, if core devs want to change any of the styling, etc, please just go ahead.

### CODE REVIEW

**Comment by mattpap** [comment]
>  `examples/integration`

Given those are very nice examples, I would put those under `examples/plotting/file`. Besides this, `examples/integration` is deprecated and any "examples" (really poor man's visual tests) in that directory will be moved to bokehjs' tests/examples.

**Comment by mattpap** [comment]
Also, if you have some spare time, maybe you could submit bokehjs version of those examples (under `bokehjs/examples`; as a separate PR)? bokehjs' examples really lagging behind bokeh.

**Comment by ianthomas23** [comment]
> Given those are very nice examples, I would put those under `examples/plotting/file`.

I've moved them.

**Comment by bryevdv** [comment]
@ianthomas23 These are really fantastic, thanks for putting them together! Two small requests before merging:

* Both plots: Set `y_range.start = 0` on both, to lower the baseline directly on to the x-axis
* Second plot: legend  lightly overlaps on of the tall bars (could move legend to left side or shrink font?)

And a few comments for optional consideration or possibly future:

* first plot: would the `4e-14` ticks labels look good as `4x10^{-14}` ? I think they would have to be fixes ticks for now? An issue to allow this as an automatic option would also be good to open. 
* first plot: could consider annotating temps with `LabelSet` labels next to curves instead of in a legend
*  complex fractions are a bit hard to parse in vertical orientation. Once `Div` is possible, we could instead add divs above the plots with a sentence or two of context, including the full formulas. I.e. in the first case the axis would remain `B_\nu(\nu, T) \quad (J s m^{-3})` which is still a great demo of axis capability!

**Comment by ianthomas23** [comment]
>     * Both plots: Set `y_range.start = 0` on both, to lower the baseline directly on to the x-axis
Done. 
>     * Second plot: legend  lightly overlaps on of the tall bars (could move legend to left side or shrink font?)
Made the second plot slightly wider to avoid the problem.

>     * first plot: would the `4e-14` ticks labels look good as `4x10^{-14}` ? I think they would have to be fixes ticks for now? An issue to allow this as an automatic option would also be good to open.
Yes, that looks better but the labels take up a lot of space on the left. So I've moved the powers of 10 into the axis labels (in multiples of 3) and it looks cleaner.
 
>     * first plot: could consider annotating temps with `LabelSet` labels next to curves instead of in a legend
I've left this for the moment as I can't see how to avoid the lower of the `LabelSet` labels from overlapping each other without a lot of manual tweaking.

>     * complex fractions are a bit hard to parse in vertical orientation. Once `Div` is possible, we could instead add divs above the plots with a sentence or two of context, including the full formulas. I.e. in the first case the axis would remain `B_\nu(\nu, T) \quad (J s m^{-3})` which is still a great demo of axis capability!
Oh, the vertical equations are horrible and a scientist would never do this! But that is what we have to play with. Yes, they should be in a separate div, or the legend perhaps, when we can support this.

**Comment by ianthomas23** [comment]
Updated PNGs:
blackbody
normal

**Comment by bryevdv** [comment]
> I've left this for the moment as I can't see how to avoid the lower of the LabelSet labels from overlapping each other without a lot of manual tweaking.

It's probably a moral failing but I would just leave them off the lowest two and let the pattern of 500k increments speak for itself (or note it somehow in the title) 🤣 

Thanks for the updates! I have proposed delaying release to Monday to not rush the "convenience" PR and also to (hopefully) get `Div` in as well. I will go ahead and update the blog with this images for the time being though, they look great!!

**Comment by bryevdv** [comment]
I am actually going to merge this as-is and make some updates in #11599

**Review by bryevdv — APPROVED** [review]
